tool/inputs/interfaces.dart

```diff
@@ -112,13 +112,13 @@ final windowsRuntimeTypesToGenerate = [
   // changes to the generation script.
   // 'Windows.Foundation.IAsyncAction',
   // 'Windows.Foundation.IAsyncOperation`1',
+  // 'Windows.Globalization.ICalendar',
```

Comment:
`ICalendar` should not be autogenerated until we update the generation script to expose `IVectorView` types as `List`.